Cairo Desktop/CairoDesktop.Configuration/Properties/Settings.settings

```diff
@@ -98,5 +98,8 @@
     <Setting Name="FullWidthTaskBar" Type="System.Boolean" Scope="User">
       <Value Profile="(Default)">False</Value>
     </Setting>
+    <Setting Name="RunAtLogOn" Type="System.Boolean" Scope="User">
```

Comment:
This should not be a saved setting, remove this and instead read from the system registry.